modules/nextflow/src/main/groovy/nextflow/extension/PublishOp.groovy

```diff
@@ -242,7 +217,7 @@ class PublishOp {
             else {
                 log.warn "Invalid extension '${ext}' for index file '${indexPath}' -- should be CSV, JSON, or YAML"
             }
-            session.notifyFilePublish(new FilePublishEvent(null, indexPath, indexOpts.labels))
+            session.notifyFilePublish(new FilePublishEvent(null, indexPath))
```

Comment:
I think we should publish the index file with the labels specified for the output

Reply:
Good point, now that the closure is gone it would make sense. I will add this